migrations/20170306205356_new-notifications.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable quotes, max-len */
+
+exports.up = knex =>
+  knex.schema
+    .raw(`ALTER TYPE build_notifications_type ADD VALUE 'diff-accepted'`)
+    .raw(`ALTER TYPE build_notifications_type ADD VALUE 'diff-rejected'`)
```

Comment:
I don't think so. But we can do it using check constraints instead of types. But I think it's very important to have it in database either a check constraints or a type.